draft-foudil-securitytxt.md

```diff
@@ -611,7 +703,7 @@ https://tools.ietf.org/html/draft-foudil-securitytxt-02
 ## Since draft-foudil-securitytxt-02
 - Use "mailto" and "tel" (#62)
 - Fix typo in the "Example" section (#64)
-- Clarified that the root directory is a fall back option (#72)
+- Clarified that the root directory is a fallback option (#72)
```

Comment:
I have no experience with writing RFCs and the like, so wanted to just make sure that changing the history to fix a typo is okay. It seems like a reasonable thing to do but I just want to make sure.

Reply:
That's usually fine, this section is not normative